0002-crypto-conditions/0002-crypto-conditions.md

```diff
@@ -428,89 +404,57 @@ subfulfillment
 
 ### Example {#prefix-sha-256-example}
 
-Example condition:
-
-    cc:1:25:7myveZs3EaZMMuez-3kq6u69BDNYMYRMi_VF9yIuFLc:102
-
-Example fulfillment:
-
-    cf:1:DUhlbGxvIFdvcmxkISAABGDsFyuTrV5WO_STLHDhJFA0w1Rn7y79TWTr-BloNGfiv7YikfrZQy-PKYucSkiV2-KT9v_aGmja3wzN719HoMchKl_qPNqXo_TAPqny6Kwc7IalHUUhJ6vboJ0bbzMcBwo
-
+TODO
 
 ## THRESHOLD-SHA-256 {#threshold-sha-256-condition-type}
-THRESHOLD-SHA-256 is assigned the type ID 2. It relies on the SHA-256 and THRESHOLD feature suites which corresponds to a feature bitmask of 0x09.
+THRESHOLD-SHA-256 is assigned the type ID 2. It relies on the availability of the SHA-256 digest algorithm and is a compound crypto-condition type.
+
+### Condition Format {#threshold-sha-256-condition-type-condition}
 
-### Condition {#threshold-sha-256-condition-type-condition}
 The fingerprint of a THRESHOLD-SHA-256 condition is the SHA-256 digest of the fingerprint contents given below:
 
     ThresholdSha256FingerprintContents ::= SEQUENCE {
-      threshold INTEGER (0..4294967295),
-      subconditions SEQUENCE OF ThresholdSubcondition
+      threshold INTEGER (1..255),
+      subconditions SEQUENCE OF Condition,
     }
 
-    ThresholdSubcondition ::= SEQUENCE {
-      weight INTEGER (0..4294967295),
-      condition Condition
-    }
-
-The list of conditions is sorted first based on length, shortest first. Elements of the same length are sorted in lexicographic (big-endian) order, smallest first.  
+The list of sub-conditions is sorted first based on length, shortest first. Elements of the same length are sorted in lexicographic (big-endian) order, smallest first.  
```

Comment:
Can you elaborate? Does the sorting take place on the binary encoded version of the conditions?